common/djangoapps/entitlements/models.py

```diff
@@ -320,6 +329,99 @@ def get_active_entitlements_for_user(cls, user):
             enrollment_course_run=None
         ).select_related('user').select_related('enrollment_course_run')
 
+    @classmethod
+    def get_fulfillable_entitlements(cls, user):
```

Comment:
Let me know if I'm being too nitpicky about naming / docstrings, but I think this method could be more clearly named as `get_all_fulfillable_entitlements_for_user`. I understand as well, though, if that seems too verbose. Let me know what you think.

Reply:
I think the method names are getting a bit long and including all is probably not necessary here.

Reply:
That's fair. Thanks for the feedback :)